internal/frontend/http/locales/active.en.yaml

```
@@ -178,6 +178,33 @@
 - id: customization.overlay.skip
   translation: "If unsure, you can skip this step."
 
+- id: customization.bootloader.title
+  translation: "Bootloader"
+
+- id: customization.bootloader.auto.name
+  translation: "auto"
+
+- id: customization.bootloader.auto.description
+  translation: "Use the preferred bootloader that Talos imager prefers"
```

```suggestion
  translation: "Automatic bootloader selection."
```

I think `Talos imager` is something that people don't have know about.